src/org/usfirst/frc2084/CMonster2017/RobotMap.java

```diff
@@ -35,108 +44,149 @@
-    public static final double DRIVE_SUBSYSTEM_BELT_RATIO = 42 / 39;
-    public static final double DRIVE_SUBYSTEM_ENCODER_DISTANCE_PER_PULSE =
-            (Math.PI * DRIVE_SUBSYSTEM_WHEEL_DIAMETER) / 2048 / DRIVE_SUBSYSTEM_BELT_RATIO;
+	public static AHRS ahrs; // navX
+
+	public static CANTalon driveBaseLeftTalon1; // main
+	public static CANTalon driveBaseRightTalon1; // main
+	public static CANTalon driveBaseLeftTalon2; // followers
+	public static CANTalon driveBaseRightTalon2; // followers
+
+	public static CANTalon driveBaseClimberTalon; // Talon for the climber
+	public static CANTalon lifterTalon; // talon for the climber
+
+	/**
+	 * .0 Wheel diameter in feet.
+	 */
+	public static final double DRIVE_SUBSYSTEM_WHEEL_DIAMETER = 0.33333333333; 
+	//0.3333333 is 1/3 of a foot, or 4 inches (the diameter of the wheels)
 	
-    
-    /**
-     * The navX Micro used for autonomous and aiming
-     */
-    
-
-    
-
-    public static void init() {
-        // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
-        driveBaseLeftTalon1 = new Talon(0);
-        LiveWindow.addActuator("DriveBase", "LeftTalon1", (Talon) driveBaseLeftTalon1);
-        
-        driveBaseLeftEncoder = new Encoder(19, 20, false, EncodingType.k4X);
-        LiveWindow.addSensor("DriveBase", "LeftEncoder", driveBaseLeftEncoder);
-        driveBaseLeftEncoder.setDistancePerPulse(1.0);
-        driveBaseLeftEncoder.setPIDSourceType(PIDSourceType.kRate);
-        driveBaseLeftPIDController1 = new PIDController(0.1, 0.0, 0.0, 0.0, driveBaseLeftEncoder, driveBaseLeftTalon1, 0.02);
-        LiveWindow.addActuator("DriveBase", "LeftPIDController1", driveBaseLeftPIDController1);
-        driveBaseLeftPIDController1.setContinuous(false);
-        driveBaseLeftPIDController1.setAbsoluteTolerance(0.2);
-
-        driveBaseLeftPIDController1.setInputRange(-1.0, 1.0);
-        driveBaseLeftPIDController1.setOutputRange(-1.0, 1.0);
-        driveBaseRightTanlon1 = new Talon(3);
-        LiveWindow.addActuator("DriveBase", "RightTanlon1", (Talon) driveBaseRightTanlon1);
-        
-        driveBaseRightEncoder = new Encoder(22, 23, false, EncodingType.k4X);
-        LiveWindow.addSensor("DriveBase", "RightEncoder", driveBaseRightEncoder);
-        driveBaseRightEncoder.setDistancePerPulse(1.0);
-        driveBaseRightEncoder.setPIDSourceType(PIDSourceType.kRate);
-        driveBaseRightPIDController1 = new PIDController(0.1, 0.0, 0.0, 0.0, driveBaseRightEncoder, driveBaseRightTanlon1, 0.02);
-        LiveWindow.addActuator("DriveBase", "RightPIDController1", driveBaseRightPIDController1);
-        driveBaseRightPIDController1.setContinuous(false);
-        driveBaseRightPIDController1.setAbsoluteTolerance(0.2);
-
-        driveBaseRightPIDController1.setInputRange(-1.0, 1.0);
-        driveBaseRightPIDController1.setOutputRange(-1.0, 1.0);
+	public static final double DRIVE_SUBSYSTEM_BELT_RATIO = 1 / 1; // no belt
```

Comment:
This is integer division so the numbers are rounded down to the nearest integer. You got lucky here.